manifests/agent.pp

```diff
@@ -179,6 +184,7 @@
   $server                                              = $zabbix::params::agent_server,
   $listenport                                          = $zabbix::params::agent_listenport,
   $listenip                                            = $zabbix::params::agent_listenip,
+  Optional[Integer] $statusport                        = $zabbix::params::agent_statusport,
```

Comment:
Can you add the declaration of the valid interval?
Optional[Integer[1024,32767]]